plugin/src/main/java/io/jenkins/plugins/coverage/model/CoverageNode.java

```diff
@@ -255,6 +255,29 @@ public void add(final CoverageLeaf leaf) {
         leaves.add(leaf);
     }
 
+    /**
+     * Removes this node from the coverage tree.
+     */
+    void remove() {
+        // TODO: remove this method when unique paths are handled correctly
+        if (hasParent()) {
+            getParent().getChildren().remove(this);
```

Comment:
`getChildren` is currently broken and will be readonly in a future version of the API.
```suggestion
            getParent().children.remove(this);
```